x/pep/types/message_get_private_keyshares.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var _ sdk.Msg = &MsgGetPrivateKeyshares{}
+
+func NewMsgGetPrivateKeyshares(creator string, reqId string, pubkey string) *MsgGetPrivateKeyshares {
+	return &MsgGetPrivateKeyshares{
+		Creator:      creator,
+		ReqId:        reqId,
+		Rsa_64Pubkey: pubkey,
+	}
+}
+
+func (msg *MsgGetPrivateKeyshares) ValidateBasic() error {
```

Comment:
ValidateBasic can include validating the pubkey

Reply:
added validation